appside/model/dbt_worksheet_model.js

```diff
@@ -291,6 +319,7 @@ class DbtWorksheetModelConfig {
         this.section = section;
         this.info_sheet_links = false;
         this.offer_ideas = false;
+        this.pre_post_measurement = true;
```

Comment:
new feature should be disabled by default.
(after changing this to false, will need to update your tests to manually set it to true.)

Reply:
fixed. Causes code block for test_dbt_worksheet_model though.